### PR TITLE
new device mode blog post

### PR DESCRIPTION
**Pull request by pbakaus**
New blog post about the new Device Mode. Please review @paulirish and @kaycebasques!

### CODE REVIEW

**Comment by kaycebasques** [comment]
@crhym3 when I go to https://pr-2704-dot-weasel-dot-web-central.appspot.com/web/updates/devtools/?hl=en and then click on the new blog post ("A new device mode for a mobile-first generation") it redirects me to web-central.appspot.com. I haven't dug into it so it may just be the way that the page is coded, but I suspect it's related to the builder

**Comment by kaycebasques** [comment]
@pbakaus what is the color value for the blue that you are using in the annotated screenshots? No issue, just want to make sure I use the same value.

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+<img src="/web/updates/images/2016/03/device-mode-v2/select-device.png" alt="Device Mode dropdown" style="float: right;max-width: 33%;margin: 0 0 10px 10px;">
+
+The main DevTools toolbar now expands to the left side of the browser window and 
+includes the most important tools to emulate a variety of mobile and Desktop 
+devices. You can choose between two development modes:
+
+* **Responsive**
+* **Specific Device**
+
+In both modes, the viewport sits in its own resizable window within Chrome.This
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: s/Chrome.This/Chrome. This/

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+tags:
+  - devtools
+  - update
+  - chrome49
+featured_image: /web/updates/images/2016/03/device-mode-v2/device-mode-initial-view.png
+---
+
+# A new Device Mode for a mobile-first generation
+
+We've introduced the Device Mode, a way to emulate devices and work with your
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: We introduced Device Mode, a way to emulate devices and work with responsive designs, a bit more than a year ago. Now it's time for its first major upgrade, starting in Chrome 49. So, what's new?

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+featured_image: /web/updates/images/2016/03/device-mode-v2/device-mode-initial-view.png
+---
+
+# A new Device Mode for a mobile-first generation
+
+We've introduced the Device Mode, a way to emulate devices and work with your 
+responsive design, a bit more than a year ago, and it's now time for its first 
+major upgrade, starting in Chrome 49. So what's new?
+
+While we offered ways to emulate mobile in the past, the development default has
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: **Mobile is becoming the starting point in Chrome DevTools**. While we offered ways to emulate mobile in the past, the development default was desktop. Mobile emulation always had to be turned on. Now that consumption of mobile sites has overtaken desktop in many places, we're switching our position in DevTools as well.

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+whenever possible. These new composite options make it a lot easier to switch 
+between modes. To toggle certain controls or customize your experience of the 
+toolbar, hit up the little three dot menu icon.
+
+## Responsive by default
+
+<img src="/web/updates/images/2016/03/device-mode-v2/select-device.png" alt="Device Mode dropdown" style="float: right;max-width: 33%;margin: 0 0 10px 10px;">
+
+The main DevTools toolbar now expands to the left side of the browser window and 
+includes the most important tools to emulate a variety of mobile and Desktop
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: lowercase 'desktop' throughout the article

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+viewport to its size. This becomes useful when you want to get in final fixes 
+and touches for a few popular devices near the launch. Which is why we're not 
+just showing a huge list of all sorts of devices in the dropdown, but the 
+currently most popular ones. If you select one, we do our best to make it behave 
+as closely as the real deal: Touch events, user agent, viewport and device 
+chrome and UI (if available) are all emulated.
+
+## Integrated Remote Debugging
+
+Emulation, even the best one available, can only get you so far. There are
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: Emulations, even the best ones available, can only get you so far. There are simply things that emulations can't do today, like:

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+currently most popular ones. If you select one, we do our best to make it behave 
+as closely as the real deal: Touch events, user agent, viewport and device 
+chrome and UI (if available) are all emulated.
+
+## Integrated Remote Debugging
+
+Emulation, even the best one available, can only get you so far. There are 
+simply things that emulation can't do today, like:
+
+* Check if a button is large enough for your thumb
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: end all these with periods

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+* Test the performance of your site on a slower phone
+* Debug random quirks and limitations of certain devices
+
+To sufficiently test all of these scenarios, you need to test, work and debug 
+using actual physical devices.
+
+<img src="/web/updates/images/2016/03/device-mode-v2/inspect-devices.png" alt="Inspect Devices dialog">
+
+For a while now, you could browse to chrome://inspect, connect your device over
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: format `chrome://inspect` as inline code

File: src/content/en/updates/posts/2016/03/device-mode-v2.markdown [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+To sufficiently test all of these scenarios, you need to test, work and debug 
+using actual physical devices.
+
+<img src="/web/updates/images/2016/03/device-mode-v2/inspect-devices.png" alt="Inspect Devices dialog">
+
+For a while now, you could browse to chrome://inspect, connect your device over 
+USB and open a remote debugging session via DevTools. But we've now gone one 
+step further and refactored how remote debugging looks and behaves, embedding it 
+into the core of DevTools. Instead of browsing to another page, you can now 
+access "**Inspect Devices**" as a dialog directly within the new main menu. This
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2016/03/device-mode-v2.markdown]: drop the quotes around **Inspect Devices**

**Comment by crhym3** [comment]
@kaycebasques looking at [src/jekyll/_layouts/updates/index.liquid](https://github.com/google/WebFundamentals/blob/0fc6c22f0795b745b6b4b8b9154fbda7bfc10723/src/jekyll/_layouts/updates/index.liquid#L19), it generates cards with `linkHref=article.canonical_url` and [canonical_url](https://github.com/google/WebFundamentals/blob/43ecb93d74bf5ad9f038076ea19b498808beee63/src/jekyll/_plugins/pages/WFPage.rb#L396) is, if I'm not mistaken:

```
site.config['WFAbsoluteUrl'] + getFilteredUrl()
```

where `WFAbsoluteUrl` is a hard-coded value for jekyll config.

**Comment by kaycebasques** [comment]
@pbakaus please address my comments then LGTM

**Comment by kaycebasques** [comment]
OK thanks @crhym3

**Comment by crhym3** [comment]
Here's direct link to that article: https://pr-2704-dot-weasel-dot-web-central.appspot.com/web/updates/2016/03/device-mode-v2

It would be nice if `WFAbsoluteUrl` could be changed dynamically, e.g. from cmd line.

**Comment by pbakaus** [comment]
All changes addressed, good to merge. @petele what's up with that red scary looking label?

**Comment by petele** [comment]
@pbakaus - used to make sure things that aren't ready aren't merged. In the future, please remove it when it's good to go!

**Comment by pbakaus** [comment]
@petele got it thanks, will do!

**Pull request merged**